src/vendors/Datadog.Trace.Vendoring/Datadog.Trace.Vendoring.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Datadog.Trace.Vendoring</RootNamespace>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net46;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.0</TargetFrameworks>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Should we just use `<LangVersion>latest</LangVersion>` going forwards? I thought about removing it entirely but the default is `7.3` when targeting anything but `netcoreapp3.0`, which defaults to `8.0`.

[reference](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version)

Reply:
This seems like something that could be grouped into another set of changes, but shouldn't removing the `<LangVersion>` property work?